src/test/java/org/springframework/cloud/deployer/spi/mesos/marathon/MesosAppDeployerIntegrationTests.java

```diff
@@ -47,6 +50,18 @@ protected AppDeployer appDeployer() {
 	}
 
 	@Override
+	protected String randomName() {
+		return super.randomName().toLowerCase(); // Marathon wants lower-case ids
+	}
+
+	@Test
+	@Override
+	@Ignore("Needs to be implemented")
```

Comment:
@trisberg This is not used in SCDF, but is theoretically in the deployer contract